programs/server/PostgreSQLHandler.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+#include <Common/CurrentMetrics.h>
+#include <Core/PostgreSQLProtocol.h>
+#include <Poco/Net/TCPServerConnection.h>
+#include "IServer.h"
+
+#if USE_SSL
+#   include <Poco/Net/SecureStreamSocket.h>
+#endif
+
+namespace CurrentMetrics
+{
+    extern const Metric PostgreSQLConnection;
+}
+
+namespace DB
+{
+
+using namespace DB::PostgreSQLProtocol;
```

Comment:
Please don't use `using namespace`.
https://clickhouse.tech/docs/en/development/style/

Reply:
Ok, will be fixed.